parser.go

```diff
@@ -100,6 +100,12 @@ func moveToRoom(exit string) {
 		fmt.Printf("Make sure you CLIMB DOWN before you try to go anywhere!\n")
 		return
 	}
+
```

Comment:
Some context would be helpful when the player tries to leave the yard. Something like "dude, you better do something about that angry eagle before you try to make for an exit"